package.json

```diff
@@ -106,6 +116,7 @@
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-react": "7.14.3",
     "inquirer": "6.5.1",
+    "instantsearch-e2e-tests": "algolia/instantsearch-e2e-tests#v1.1.0",
```

Comment:
dumb question, since this is a private repository, is this going to break the dev environment of external contributors?

Reply:
The repository is public, the package is just not published on npm. It should work fine for external contributors.